### ISSUE TITLE
allow user to hide pipeline representation after successful completion?

### ISSUE THREAD

**Issue by adf-ncgr** [legumeinfo/lis_context_viewer#272]
I'm liking the pipelines far more than I thought I would! Despite that, however, I suspect that when taking screenshots and the like a user like myself will want to minimize their real estate/visual impact. Not a blocker, but probably a nice to have.

**Comment by alancleary** [comment]
I agree, especially since I made the context-menu navbars full height to make room for the pipelines. I'll do some investigating into how others have solved this UI/UX problem, but only after the v2 release.

**Comment by alancleary** [comment]
I just had an idea on this could be implemented: the pipeline could be put in the [external content](https://getbootstrap.com/docs/4.6/components/navbar/#external-content) portion of the context menu's navbar. This would be shown automatically when a pipeline is running and hide automatically when the pipeline completes (only successfully?). Of course there would be a button so the user could toggle it whenever they want.
